server/src/main/java/org/elasticsearch/rest/action/admin/cluster/RestNodesHotThreadsAction.java

```diff
@@ -34,23 +34,62 @@
 
 import java.io.IOException;
 import java.util.List;
+import java.util.Locale;
 
 import static org.elasticsearch.rest.RestRequest.Method.GET;
 
-
 public class RestNodesHotThreadsAction extends BaseRestHandler {
 
+    static final String formatDeprecatedMessageWithoutNodeID = "[%s] is a deprecated endpoint. Please use [/nodes/hot_threads] instead.";
```

Comment:
These can all be private now.

Reply:
I have signed the CLA but I did not get any confirmation email or other notification to indicate that I have successfully signed it. Thanks!